CONTRIBUTING.md

```diff
@@ -88,12 +88,12 @@ The first time you run the app, simply run:
 ```
 yarn install # installs node modules
 docker-compose up # Spin up docker containers
+yarn start #Spin up websever and worker
 ```
 
-That's it.  Everything is ran in docker-compose, including redis, postgres, ngrok and the app (main and worker thread).
-If you want to debug, you can connect to the remote port of 9229 for the main thread and 9230 for the worker thread in docker.  Any changes to the code will restart the node server automatically.
+That's it.  Dependant services ran in docker-compose, including redis, postgres, ngrok and the app (main and worker thread) run in local `yarn start`.
```

Comment:
This is a bit confusing cause **redis, postgres, ngrok and the app** kind of imply that the **app** is also one of the dependant services, can we reword this sentence?
```suggestion
That's it.  Dependant services ran in docker-compose, including redis, postgres and ngrok. And now to run the app (main and worker thread), please run in local `yarn start`.
```